src/main/java/build/buildfarm/IndexWorker.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
```

Comment:
Do you think we need this tool since we are exposing this via a grpc service? Would a test be sufficient enough?

Reply:
At the very least, I should add a mock test.  My issue has been that jedis throws weird runtime exceptions when you use the library in subtly incorrect ways. For example, I couldn't iterate over the whole cluster and do a certain key operations, but instead had to iterate over the cluster nodes individually which the mock wouldn't have caught.  The closest thing we have to an integration-level (for stressing grpc/redis) is probably building rNA with dev?  

The client side tool lets me mimic bf-admin and get some more confidence before landing.
I see the concern of having a bunch of random tools that hit grpc endpoints though.  Maybe they could be consolidated better? Or if we could somehow run bf-admin locally, and evoke changes that way, I would have no need for the tools.

Reply:
I'm going to land and then followup with the other ideas of testing. We could remove this in the future.